addons/mail/static/src/core/common/message_model.js

```diff
@@ -326,6 +326,7 @@ export class Message extends Record {
                     "<div></div>",
                     "<div><br></div>",
                     "<div><br/></div>",
+                    '<divclass="o-paragraph"><br></div>', // space removed with `.replace(/\s/g, "")` and cannot switch the position of single and double quote
```

Comment:
I think we need both versions of quotes (prod as double but tests have single).

Reply:
I agree that we should create a new method in this case. But I think the method should still using `textContent` as it is the most reliable way to check if there's any content in the node. The only thing we need to do is to have extra query if there's img or video or audio in the html. If there are more common used tags taken into consider, we can add them later.

So what I propose:
1. rename `isHtmlTextContentEmpty` (or not in thsi PR, tbd) and create a new `isHtmlEmpty` (or whatever name)
2. the new method calling the old one and extra query if there's img videos audios inside.